samples/cpp/classification_sample_async/main.cpp

```diff
@@ -63,8 +64,8 @@ bool ParseAndCheckCommandLine(int argc, char* argv[]) {
 
 int main(int argc, char* argv[]) {
     try {
-        // -------- Get OpenVINO Runtime version --------
-        slog::info << "OpenVINO runtime: " << ov::get_openvino_version() << slog::endl;
+        // -------- Get OpenVINO runtime version --------
+        slog::info << ov::get_openvino_version() << slog::endl;
```

Comment:
If I remember right here we just print the pointer to Vestion structure 😄

Reply:
to not print pointer appropriate operator<< added. Also, it might make sense to take changes from #8687 here